docs/INSTALL.md

```diff
@@ -150,6 +150,24 @@ source $HOME/fprime-venv/bin/activate
 If installing without a virtual environment, PIP occasionally uses `$HOME/.local/bin` as a place to install user tools.
 Users running without virtual environments should add this directory to the path.
 
+### fprime-util generate: Failed to run fpp-locate-deps
```

Comment:
I agree, we can make the documentation much more terse this way.

Fixed.

Reply:
Apologies, breezed through the first half of that, so I can open this again in case you have any other comments.

In my experience as a user of FPrime thus far, it's not intuitive that CMake is calling a helper script and where exactly that helper script is within the CMake build process.

I would vote for keeping this section.